internal/db/modelmigration_test.go

```diff
@@ -75,7 +89,7 @@ func (s *dbSuite) TestGetModelMigration(c *qt.C) {
 	c.Assert(s.Database.DB.Create(&migration).Error, qt.IsNil)
 
 	lookup := dbmodel.IncomingModelMigration{ModelUUID: migration.ModelUUID}
-	err = s.Database.GetIncomingModelMigration(context.Background(), &lookup)
+	err := s.Database.GetIncomingModelMigration(context.Background(), &lookup)
```

Comment:
Please check it looks as intended here, we're just checking err but perhaps it actually gotten nothing